src/components/rtc-settings-panel.tsx

```diff
@@ -143,10 +144,6 @@ export function RTCSettingsPanel() {
 						{ __( 'Notifications', 'vip-real-time-collaboration' ) }
 					</Heading>
 
-					<Heading level={ 2 } style={ { marginBottom: '2px' } }>
```

Comment:
Why was this removed?

Reply:
I thought it was a mistake. It's a level 2 heading following a level 3 and looked anomalous. It's also the only sub-section so provided no distinction.

Reply:
It was done on purpose, to allow for the notifications to be categorized based on their purpose. I think the User sub-section was removed? 

There were supposed to be 2 of em:

- Post: This would be for save/publish and any other post specific action
- Collaborator: This would be for when a collaborator enters/exits a post

I think because the second one was removed it looks confusing. Lemme dig that up, and re-add it because the toggles don't make a lot of sense right now. For example, "enters" doesn't quite convey what that means.

Reply:
It was removed in https://github.com/Automattic/vip-real-time-collaboration/commit/7eca8cda759b7cf6d8008bf67e4687215ea60475.

I'll add it back in.

Reply:
Can you fix the heading levels?

Reply:
Noting that this has been fixed in #120